networkit/cpp/centrality/GroupHarmonicClosenessImpl.hpp

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
After the .cpp file is merged into this one, I would suggest to merge both into `GroupHarmonicCloseness.cpp`. Then the entire template can be put into an anonymous namespace so that it is not visible outside of `GroupHarmonicCloseness.cpp`.